duet/mr1DEVELOPMENT/views/dev_desktop_dates_partners.view.lkml

```diff
@@ -0,0 +1,65 @@
+view: dev_desktop_dates_partners {
```

Comment:
Is this really necessary, wouldn't it be a fair assumption to make there will be a date for each date starting `2021-01-01`? Why this additional artifact?